qa/tasks/cephfs/test_volumes.py

```diff
@@ -302,6 +305,30 @@ def test_volume_rm(self):
         else:
             raise RuntimeError("expected the 'fs volume rm' command to fail.")
 
+    def test_volume_rm_when_mon_delete_pool_false(self):
+        """
+        That the volume can only be removed when --yes-i-really-mean-it is used
```

Comment:
This doc string is same as that of the test above. But doesn't this one actually test the effect of 'mon_allow_pool_delete' mon config setting on the volume deletion.

Reply:
Done